b2luigi/batch/processes/gbasf2.py

```diff
@@ -0,0 +1,782 @@
+import hashlib
+import json
+import os
+import shlex
+import shutil
+import subprocess
+import tempfile
+import warnings
+from collections import Counter
+from datetime import datetime, timedelta
+from functools import lru_cache
+
+from luigi.target import Target
+from b2luigi.basf2_helper.utils import get_basf2_git_hash
+from b2luigi.batch.processes import BatchProcess, JobStatus
+from b2luigi.core.settings import get_setting
+from b2luigi.core.utils import get_log_file_dir, get_task_file_dir
+from jinja2 import Template
+
+import basf2.pickle_path as b2pp
```

Comment:
Ah, sorry for the late comment: can you please make this a lazy import? Otherwise the core components of `b2luigi` can not be imported without having `basf2` - which also means all the tests fail.

Reply:
Done in 16c9d12.